tensorflow/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -63,6 +64,13 @@ cublasLtEpilogue_t AsCublasLtEpilogue(BlasLt::Epilogue epilogue) {
       return CUBLASLT_EPILOGUE_BIAS;
     case BlasLt::Epilogue::kBiasThenReLU:
       return CUBLASLT_EPILOGUE_RELU_BIAS;
+    case BlasLt::Epilogue::kBiasThenGeLUApproximate:
```

Comment:
enumeration value 'kGeLU' not handled in switch

Reply:
Resolved. PTAL.